db/seeds.rb

```diff
@@ -16,22 +16,150 @@
+		"Focus on structured breathing.", 
+		"Call a close friend.", 
+		"Immerse yourself in a hobby you enjoy.", 
+		"Work out.", 
+		"Listen to soothing music.", 
+		"Listen to nature sounds.", 
+		"Go for a run.", 
+		"Take a short nap.", 
+		"Go for a bike ride.", 
+		"Medidate.", 
+		"Go for a walk.", 
+		"Have a conversation with friends or family.", 
+		"Go outside.", 
+		"Have a glass of water.", 
+		"Play with your pet, or visit an animal shelter.", 
+		"Picture a relaxing scene.", 
+		"Sit in the sun for 15 minutes."
+	],
+	anger: [
+		"Listen to your favorite song.", 
+		"Take a moment to be present in the moment.", 
+		"Write a letter you do not intend to send.", 
+		"Take a time out.", 
+		"Go for a walk.", 
+		"Take deep breaths and think about something that makes you happy.", 
+		"Go for a bike ride.", 
+		"Go for a run.", 
+		"Talk with friends or family.", 
+		"Get some fresh air.", 
+		"Go to the gym.", 
+		"Go to the movies."
+	],
+	fear: [
+		"Text a friend.", 
+		"Focus on the present moment.", 
+		"Focus on your breathing.", 
+		"Seek out support.", 
+		"Go for a walk.", 
+		"Watch a comforting movie.", 
+		"Meditate.", 
+		"Exercise.", 
+		"Talk to someone you trust.", 
+		"Change your surroundings.", 
+		"Listen to music.", 
+		"Take a deep breath.", 
+		"Share your feelings with a close friend."
+	],
+	joy: [
+		"Go for a walk.", 
+		"Call a friend.", 
+		"Make someone laugh.", 
+		"Hug a loved one.", 
+		"Exercise.", 
+		"Smile at a stranger.", 
+		"Remain in the present.", 
+		"Go outside.",
+		"Mow your lawn."
+	],
+	sadness: [
+		"Do some push-ups.", 
+		"Spend some time with your pet, or visit an animal shelter.", 
+		"Seek out support.", 
+		"Write down your thoughts.", 
+		"Take time to get out of the house.", 
+		"Breathe.", 
+		"Spend some time outside.", 
+		"Take a short nap.", 
+		"Drink a cup of tea.", 
+		"Watch a funny movie.", 
+		"Talk to family or friends.", 
+		"Focus your enery on work or a hobby.", 
+		"Spend some time to take care of an easy task."
+	],
+	anxious: [
+		"Eat a burrito.",
+		"Focus on breathing.", 
+		"Have a cup of tea.", 
+		"Speak with someone you trust.", 
+		"Seek out a friend or loved one.", 
+		"Go for a walk.", 
+		"Treat yourself to a snack that you enjoy.", 
+		"Meditate.", 
+		"Take a deep breath.", 
+		"Stretch.", 
+		"Go for a run.", 
+		"Listen to relaxing music.", 
+		"Make a list."
+	],
+	disgust: [
+		"Spend a moment to consider the root of your current emotional state.", 
+		"Go for a walk.", 
+		"Try to use your empathy.", 
+		"Express your feelings in a constructive way.", 
+		"Remove yourself from your current surroundings.",
+		"Have some ice cream."
+	],
+	tired: [
+		"Get up and move around.", 
+		"Drink water.", 
+		"Stay active.", 
+		"Go for a walk.", 
+		"Get a cup of coffe.", 
+		"Have a cup of tea.", 
+		"Tell a joke.", 
+		"Take a short nap.", 
+		"Meditate.", 
+		"Exercise.", 
+		"Go for a run.", 
+		"Take a break for a few moments."
```

Comment:
Delete lines 152, 153, 154, 155

Add - 
  "Did you get much sleep? Try closing your eyes for 20 minutes.",
  "Take a shower, or wash your face in cold water."